README.md

```diff
@@ -102,51 +104,36 @@ Example command:
 
 Clone this repository to your desired folder:
 
-<!--
```

Comment:
- [ ] Please as mentioned by the previous reviewer you should make sure your design sticks to the wireframe 🙃 😉 

## Users Index Page
Your design
<img width="360" alt="Screen Shot 2022-12-06 at 12 11 47" src="https://user-images.githubusercontent.com/74501523/205869153-8e091a91-1b7f-4769-b7bb-f4fc0fd016ba.png">

Wireframe
<img width="440" alt="Screen Shot 2022-12-06 at 12 05 16" src="https://user-images.githubusercontent.com/74501523/205869204-9f6622bc-8e7f-4a0f-b99c-edc99515f370.png">

## Users Show Page
Your design
<img width="360" alt="Screen Shot 2022-12-06 at 12 16 10" src="https://user-images.githubusercontent.com/74501523/205870209-5b3c09d7-9b42-4863-98f8-21bf3d56c221.png">

Wireframe
<img width="441" alt="Screen Shot 2022-12-06 at 12 16 33" src="https://user-images.githubusercontent.com/74501523/205870265-2417bda0-c24e-468b-a51f-aff95da3a6ae.png">

## Posts Index Page
Your design
<img width="362" alt="Screen Shot 2022-12-06 at 12 18 59" src="https://user-images.githubusercontent.com/74501523/205870949-e3b16031-1655-40c9-b983-6f058516f4e1.png">

Wireframe
<img width="444" alt="Screen Shot 2022-12-06 at 12 19 59" src="https://user-images.githubusercontent.com/74501523/205870987-97c77fb5-9b1b-48a0-8a1c-ced488c4d2d8.png">

## Post Show Page
Your design
<img width="363" alt="Screen Shot 2022-12-06 at 12 22 22" src="https://user-images.githubusercontent.com/74501523/205871522-0ef62f0a-87e2-43ed-9990-52c1ec819b43.png">

Wireframe
<img width="445" alt="Screen Shot 2022-12-06 at 12 22 41" src="https://user-images.githubusercontent.com/74501523/205871555-5109660d-a383-491f-b331-1da4c651e774.png">

- [ ] Please kindly fix your design 🙏

Reply:
Thank you @cynthiainga . Please can you explicit as what or where it doesn't stick to the wireframe

Reply:
- [ ] Hey @t-bello7, as you mentioned you should also make sure you have the border lines and that border lines are aligned correctly as in the wireframe 👍 👍

Reply:
Okay Thanks.